src/client/results/journalism/entities/PlayerInformation.js

```diff
@@ -1,9 +1,134 @@
 import React from 'react';
+import moment from 'moment';
+import { Statistic } from '../../../common/common';
 
-const PlayerInformation = () => (
+const tryPropertyOrElse = (object, property, else_) => {
+  try {
+    return object[property].value;
+  } catch (e) {
+    return else_;
+  }
+}
+
+const tryPropertyOrNA = (object, property) => (
```

Comment:
probably would move this into utilities and add a small test... but not going to ask you to do that if you cba for this project X)